src/Symfony/Component/Config/Tests/Loader/DelegatingLoaderTest.php

```diff
@@ -80,4 +80,21 @@ public function testLoadThrowsAnExceptionIfTheResourceCannotBeLoaded()
 
         $loader->load('foo');
     }
+
+    /**
+     * @expectedException \Symfony\Component\Config\Exception\FileLoaderLoadException
+     */
+    public function testWithLoadingSetToTrue()
+    {
+        $loader = $this->getMock('Symfony\Component\Config\Loader\LoaderInterface');
+        $loader->expects($this->any())->method('supports')->will($this->returnValue(true));
+        $resolver = new LoaderResolver(array($loader));
+        $loader = new DelegatingLoader($resolver);
+
+        $loading = new \ReflectionProperty($loader, 'loading');
```

Comment:
Can we please create a real test that covers functionality instead of implementation?